src/QueryRunner.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SynapseTransformation;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
+use Keboola\SynapseTransformation\Configuration\Code;
+use Keboola\SynapseTransformation\Configuration\Script;
+use Keboola\SynapseTransformation\Configuration\Block;
+use Keboola\SynapseTransformation\Exception\UserException;
+use Psr\Log\LoggerInterface;
+
+class QueryRunner
+{
+    private LoggerInterface $logger;
+
+    private Connection $connection;
+
+    private QueryFormatter $queryFormatter;
+
+    public function __construct(LoggerInterface $logger, Connection $connection, QueryFormatter $queryFormatter)
+    {
+        $this->logger = $logger;
+        $this->connection = $connection;
+        $this->queryFormatter = $queryFormatter;
+    }
+
+    /**
+     * @param array|Block[] $blocks
+     */
+    public function processBlocks(array $blocks): void
+    {
+        foreach ($blocks as $block) {
+            $this->processBlock($block);
+        }
+    }
+
+    private function processBlock(Block $block): void
+    {
+        $this->logger->info(sprintf('Processing block "%s".', $block->getName()));
+        foreach ($block->getCodes() as $code) {
+            $this->processCode($block, $code);
+        }
+    }
+
+    private function processCode(Block $block, Code $code): void
+    {
+        $this->logger->info(sprintf('Processing code "%s".', $code->getName()));
+        foreach ($code->getScripts() as $script) {
+            $this->processScript($block, $code, $script);
+        }
+    }
+
+    private function processScript(Block $block, Code $code, Script $script): void
+    {
+        // Remove comments
+        $sql = $this->queryFormatter->removeComments($script->getRawSql());
+        $sqlToLog = $this->queryFormatter->formatToLog($sql);
+
+        // Skip empty query
+        if (!$sql) {
+            $this->logger->info('Ignoring empty query.');
```

Comment:
toto bych mozna vyhodil - ono to bude asi produkovat misto kazdyho komenentare a zaplevelovat log

Reply:
ok. vyhodim.

Reply:
odstranene. https://github.com/keboola/synapse-transformation/pull/1/commits/f9ab49e88a5330370858cc11dd695a81cc40383f